pattern-library/js/edx-icons.js

```diff
@@ -1,18 +1 @@
-;(function(define) {
-    'use strict';
-    define(['afontgarde'], function() {
-        // support icons that aren't yet using UXPL
-        window.AFontGarde('FontAwesome', {
-            glyphs: '&#61515;'
-        });
-        // the UXPL icons
-        window.AFontGarde('edx-icons', {
-            glyphs: '\uE621\uE622\uE623'
-        });
-    });
-}).call(
-    this,
-    typeof define === 'function' && define.amd ? define :
-        (typeof RequireJS !== 'undefined' ? RequireJS.define :
-            edx.GlobalLoader.defineAs('edxicons', 'edx-pattern-library/js/edx-icons.js'))
-);
+require('font-awesome-loader');
```

Comment:
I think you can just remove this file, and change the pldoc to have this dependency instead. It doesn't make sense to have this file here because it will be included in the npm package, but clients who use the Pattern Library might not be using Webpack.